raisinbread/RB_files/RB_test_battery.sql

```diff
@@ -1,12 +1,12 @@
 SET FOREIGN_KEY_CHECKS=0;
 TRUNCATE TABLE `test_battery`;
 LOCK TABLES `test_battery` WRITE;
-INSERT INTO `test_battery` (`ID`, `Test_name`, `AgeMinDays`, `AgeMaxDays`, `Active`, `Stage`, `SubprojectID`, `Visit_label`, `CenterID`, `firstVisit`, `instr_order`) VALUES (131,'aosi',1,2147483647,'Y','Visit',NULL,'V1',NULL,NULL,NULL);
-INSERT INTO `test_battery` (`ID`, `Test_name`, `AgeMinDays`, `AgeMaxDays`, `Active`, `Stage`, `SubprojectID`, `Visit_label`, `CenterID`, `firstVisit`, `instr_order`) VALUES (132,'aosi',1,2147483647,'Y','Visit',NULL,'V2',NULL,NULL,NULL);
-INSERT INTO `test_battery` (`ID`, `Test_name`, `AgeMinDays`, `AgeMaxDays`, `Active`, `Stage`, `SubprojectID`, `Visit_label`, `CenterID`, `firstVisit`, `instr_order`) VALUES (133,'aosi',1,2147483647,'Y','Visit',NULL,'V3',NULL,NULL,NULL);
-INSERT INTO `test_battery` (`ID`, `Test_name`, `AgeMinDays`, `AgeMaxDays`, `Active`, `Stage`, `SubprojectID`, `Visit_label`, `CenterID`, `firstVisit`, `instr_order`) VALUES (134,'radiology_review',1,2147483647,'Y','Visit',1,'V1',NULL,NULL,NULL);
+INSERT INTO `test_battery` (`ID`, `Test_name`, `AgeMinDays`, `AgeMaxDays`, `Active`, `Stage`, `SubprojectID`, `Visit_label`, `CenterID`, `firstVisit`, `instr_order`) VALUES (131,'aosi',1,365,'Y','Visit',NULL,'V1',NULL,NULL,NULL);
+INSERT INTO `test_battery` (`ID`, `Test_name`, `AgeMinDays`, `AgeMaxDays`, `Active`, `Stage`, `SubprojectID`, `Visit_label`, `CenterID`, `firstVisit`, `instr_order`) VALUES (132,'aosi',366,730,'Y','Visit',NULL,'V2',NULL,NULL,NULL);
+INSERT INTO `test_battery` (`ID`, `Test_name`, `AgeMinDays`, `AgeMaxDays`, `Active`, `Stage`, `SubprojectID`, `Visit_label`, `CenterID`, `firstVisit`, `instr_order`) VALUES (133,'aosi',731,1460,'Y','Visit',NULL,'V3',NULL,NULL,NULL);
+INSERT INTO `test_battery` (`ID`, `Test_name`, `AgeMinDays`, `AgeMaxDays`, `Active`, `Stage`, `SubprojectID`, `Visit_label`, `CenterID`, `firstVisit`, `instr_order`) VALUES (134,'radiology_review',1,2147483647,'Y','Visit',1,'V1',2,NULL,NULL);
```

Comment:
the next tow inserts allow testing whether the instrument should be populated based on the side of the visit